Source/Core/DolphinQt/NetPlay/NetPlayDialog.cpp

```diff
@@ -458,6 +462,56 @@ void NetPlayDialog::show(std::string nickname, bool use_traversal)
   UpdateGUI();
 }
 
+void NetPlayDialog::UpdateDiscordPresence()
+{
+#ifdef USE_DISCORD_PRESENCE
+  // both m_current_game and m_player_count need to be set for the status to be displayed correctly
+  if (m_player_count == 0 || m_current_game.empty())
+    return;
+
+  const auto use_default = [this]() {
+    Discord::UpdateDiscordPresence(m_player_count, Discord::SecretType::Empty, "", m_current_game);
+  };
+
```

Comment:
A condition should be added here to call `use_default()` if the game is running, and this function additionally called on game start/stop. This can be achieved using `OnMsgStartGame` and `OnMsgStopGame`, but be sure to use `QueueOnObject` as these are called from the NetPlay thread.